core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -1193,16 +1192,14 @@ public void close() throws IOException {
 
       if (sampleReaders != null) {
         for (LocalityGroupReader lgr : sampleReaders) {
-          try {
-            lgr.close();
-          } catch (IOException e) {
-            log.warn("Errored out attempting to close LocalityGroupReader.", e);
-          }
+          lgr.close();
         }
       }
 
       try {
         reader.close();
+      } catch (IOException e) {
+        throw new RuntimeException(e);
```

Comment:
Should probably use the new `UncheckedIOException` wrapper here and elsewhere when `IOException` is wrapped.